catalog-info.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: vets-api
```

Comment:
Short name of the component. There will also be an API catalog entry that's similarly named.